### PR TITLE
DEV: LEAF-1856 - disable_scrollbar

### PR DESCRIPTION
**Pull request by Vaceek**

### CODE REVIEW

File: LEAF_Request_Portal/admin/templates/mod_form.tpl [hunk 1]
```diff
@@ -1129,20 +1129,21 @@ function getForm(indicatorID, series) {
         if(<!--{$hasDevConsoleAccess}--> == 1) {
             $('#button_advanced').css('display', 'none');
             // triggers overflow of content
-            $('#xhr').css('overflow-y', 'scroll');
+            //$('#xhr').css('overflow-y', 'scroll');
```
Comment (gsteinhoff) [inline LEAF_Request_Portal/admin/templates/mod_form.tpl:1132]: Should just delete this line if not used

File: LEAF_Request_Portal/admin/templates/mod_form.tpl [hunk 2]
```diff
@@ -1129,20 +1129,21 @@ function getForm(indicatorID, series) {
...
             alert('Notice: Please go to Admin Panel -> LEAF Programmer to ensure continued access to this area.');
             $('#button_advanced').css('display', 'none');
+			$('#advanced').css('height', 'auto');
     	    $('#advanced').css('visibility', 'visible');
         }
     });
 
     // resets overflow on new dialog open
-    $('#xhr').css('overflow-y', 'unset');
+    // $('#xhr').css('overflow-y', 'unset');
```
Comment (gsteinhoff) [inline LEAF_Request_Portal/admin/templates/mod_form.tpl:1146]: Again, not being used so should delete

**Review by gsteinhoff — CHANGES_REQUESTED** [review]

File: LEAF_Request_Portal/admin/templates/mod_form.tpl (hunk 2 above)
Comment (Vaceek) [reply LEAF_Request_Portal/admin/templates/mod_form.tpl]: items removed

**Review by pete-nerantzinis — APPROVED** [review]

**Review by gsteinhoff — CHANGES_REQUESTED** [review]
Why does this PR now have tons of extra content on it from previous commits to Dev?  This shouldn't have all of these and only the commits of your changes.  Did you merge with Dev?  These should be removed so this PR is clean of extra code not related to the initial task.

**Comment by gsteinhoff** [comment]
I say this because this task has some of my un-launched Email Mass Action stuff as well as things that I think were launched last Sprint 7.  Again, this should be a clean PR not containing other commits from other projects.

**Comment by Vaceek** [comment]
resetting my phpstorm/git caused this PR to have extra items in it in error. closing PR to clean up local

**Pull request closed without merging**